.gitattributes

```diff
@@ -8,24 +8,18 @@ tests/Report/AbstractReporterTestCase.php export-ignore
 tests/Test/AbstractFixerWithAliasedOptionsTestCase.php export-ignore
 tests/Test/AbstractTransformerTestCase.php export-ignore
```

Comment:
Could you add `tests/Test/IsIdenticalConstraint.php` and revert d481be5?

Reply:
Done

Reply:
Actually the trait is used in classes that are *not* excluded, so my comment was wrong, sorry :/

Reply:
Reverted :)